packages/material-ui/package.json

```diff
@@ -40,7 +40,6 @@
     "@material-ui/styles": "^4.0.0-alpha.3",
     "@material-ui/system": "^4.0.0-alpha.0",
     "@material-ui/utils": "^4.0.0-alpha.3",
-    "@types/jss": "^9.5.6",
```

Comment:
We're still importing it in `packages/material-ui/src/styles/withStyles.d.ts`. We should probably just kill that one or rather simply re-export `@material-ui/styles`

Reply:
`@types/jss` is not needed anymore. You're using `jss@10-alpha` version.
That's why I updated the type to `JSS. StyleSheetFactoryOptions`

Reply:
But I agree there is maybe something better to do with the types
'createStyles.d.ts', `makeStyles.d.ts` (which is not created), `withStyles.d.ts`, `withTheme.d.ts`

Removing the `@types/jss` was a good baby step to remove a typescript issue.

Reply:
Right. In other words (for future me): Since we don't use `jss@9` anymore we don't need `@types/jss` anymore since `jss@10` bundles their own.